drafts/arcalinea-tx-expiry/draft.rst

```diff
@@ -0,0 +1,83 @@
+::
+
+  ZIP: 203
+  Title: Transaction Expiry
+  Author: Jay Graber <jay@z.cash>
+  Category: Consensus
+  Created: 2018-01-09
+  License: MIT
+
+
+Abstract
+===========
+
+This is an Standards ZIP describing a new consensus rule to set an expiration time for transactions that are not mined to be removed from the mempool.
```

Comment:
s/an/a/

Reply:
"to set an expiration time after which a transaction cannot be mined. If it is not mined within that time, the transaction will be removed from nodes' mempools."